Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/NavigationBarSamplePage.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<Page x:Class="Uno.Gallery.Views.SamplePages.NavigationBarSamplePage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:Uno.Gallery"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+      mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <local:SamplePageLayout>
+            <local:SamplePageLayout.MaterialTemplate>
```

Comment:
Please use XamlDisplay for the samples in Gallery

Reply:
this is just a button to launch the actual sample page; there is no point in showing the xaml for that.